src/stores/BreadcrumbsStore.ts

```diff
@@ -84,8 +84,14 @@ export class BreadcrumbsStore extends AsyncStoreWithClient<IState> {
             } else {
                 // The tests might not result in a valid room object.
                 const room = this.matrixClient.getRoom(payload.room_id);
-                if (room) await this.appendRoom(room);
+                const roomTimeline = room?.timeline;
+                const length = roomTimeline?roomTimeline.length:0;
+                const roomInvite = length==0||(length>0&&roomTimeline[length-1].getContent().membership==="leave");
```

Comment:
This seems to only consider rooms you've left as you only check that your membership is `leave`. I suggest we only show rooms where you membership is `join`. That way invites will be excluded, same as rooms you've left.

You should probably use `room.getMyMembership()` rather than inspecting state events in the timeline